### ISSUE TITLE
Could you please provide the details of the pre-trained models?

### ISSUE THREAD

**Issue by silhouettesia** [DTaoo/VGGish#1]
Did you train the models on the AudioSet for AED? How did you get the raw audio dataset?
Could you explain how to train the model without the fc layers?
Thanks!

**Comment by silhouettesia** [comment]
or you just turn the released tensorflow trained model into keras ?

**Comment by DTaoo** [comment]
Yes, I just simply turn the released TF trained model into Keras, without any training on the AudioSet.

**Comment by xav12358** [comment]
@DTaoo  Is the repo for train the Vggish model ? Or is it to use the pretrained VGGish model?

**Comment by rola93** [comment]
I'm late for the party... but will answer it anyway, hope it's useful for someone else.

@xav12358 It is for both... you can initialize the model with random weights and retrain them from scratch, or just use the given weights, or even retrain from the given weights (aka fine tuning). This repo provides a Keras version of the VGGish model, and let you load it with the given weigths, then you can do whatever you want with it. The code for train/fine tune the model is not given but is not that difficult.

I'll try to upload one example later

@DTaoo shouldn't this issue be clossed?

**Comment by Jacob-Stevens-Haas** [comment]
It might be nice if the README.md had a link to where to find `vggish_audioset_weights.h5` and `vggish_audioset_weights_without_fc2.h5`.  The current link to TensorFlow's version points to a tensorflow checkpoint file for the weights, which can't be directly loaded into keras.
